core/src/main/java/org/apache/calcite/sql/SqlUtil.java

```diff
@@ -417,24 +418,52 @@ public static void unparseSqlIdentifierSyntax(
     writer.endList(frame);
   }
 
+  private static void unparseSetOpSyntax(
```

Comment:
It's not obvious what this method does. Can you add brief javadoc.

Reply:
Ok, I've added a doc to explain it.